fixtures/search-syntax/invalid_aggregate_column_with_duration_filter.json

```diff
@@ -2,9 +2,54 @@
   {
     "query": "avg(stack.colno):>500s",
     "result": [
-      {"type": "spaces", "value": ""},
-      {"type": "freeText", "quoted": false, "value": "avg(stack.colno):>500s"},
-      {"type": "spaces", "value": ""}
```

Comment:
We need to verify this doesn't break backend tests as well.

Needs this #36606